.github/workflows/android.yml

```diff
@@ -0,0 +1,91 @@
+name: Android
+on:
+  workflow_call:
+    inputs:
+      profile:
+        required: false
+        default: "release"
+        type: string
+  workflow_dispatch:
+    inputs:
+      profile:
+        required: false
+        default: "release"
+        type: string
+        options: ["release", "debug", "production"]
+  push:
+    branches: ["try-android"]
+
+env:
+  RUST_BACKTRACE: 1
+  SHELL: /bin/bash
+  SCCACHE_GHA_ENABLED: "true"
+  CCACHE: "sccache"
+  CARGO_INCREMENTAL: 0
+
+jobs:
+  build:
+    name: Android Build
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        arch: ['armv7-linux-androideabi', 'i686-linux-android']
+    steps:
+      - uses: actions/checkout@v3
+        if: github.event_name != 'issue_comment' && github.event_name != 'pull_request_target'
+        with:
+          fetch-depth: 2
+      # This is necessary to checkout the pull request if this run was triggered
+      # via an `issue_comment` action on a pull request.
+      - uses: actions/checkout@v3
+        if: github.event_name == 'issue_comment' || github.event_name == 'pull_request_target'
+        with:
+          ref: refs/pull/${{ github.event.issue.number || github.event.number }}/head
+          fetch-depth: 2
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.3
+      - name: Install taplo
+        uses: baptiste0928/cargo-install@v2
+        with:
+          crate: taplo-cli
+          locked: true
+      - name: Bootstrap Python
+        run: python3 -m pip install --upgrade pip virtualenv
+      - name: Bootstrap dependencies
+        run: sudo apt update && python3 ./mach bootstrap
+      - name: Build (arch ${{ matrix.arch }} profile ${{ inputs.profile }})
+        run: |
+          export JAVA_HOME="${JAVA_HOME_17_X64}"
```

Comment:
Don't we need to install a specific version of the SDK and NDK here?

Reply:
We are relying on the SDK and NDK preinstalled in the [runner image](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#environment-variables-2)

Reply:
Let's use actions like "nttld/setup-ndk" that we use for mozjs to ensure that the proper version is installed. If the runner image changes, this image this might break one day in the future.